examples/gno.land/r/nt/commondao/public.gno

```diff
@@ -46,9 +46,19 @@ func New(name string, options ...Option) *commondao.CommonDAO {
 	return createDAO(name, caller, options)
 }
 
+func UpdateUser(_ realm, daoID uint64, user std.Address) {
+	dao := mustGetDAO(daoID)
+	dao.Members().Add(user)
+}
+
+func Propose(_ realm, daoID uint64, user std.Address, pd commondao.ProposalDefinition) (*commondao.Proposal, error) {
+	dao := mustGetDAO(daoID)
+	return dao.Propose(user, pd)
+}
```

Comment:
the impact is that after this change(cross to the realm where methods are declared),  we'll need cross into the obj's storage realm first(by explicit func), then to invoke the method, which is clumsy...

So, maybe the correct path should be still soft cross into recv's storage realm, then assert the method's declaration realm equal to the storage realm, otherwise panic?

Reply:
see #4649.